addOns/ascanrulesBeta/src/test/resources/org/zaproxy/zap/extension/ascanrulesBeta/xxescanrule/ValidatedXml.txt

```diff
@@ -0,0 +1,18 @@
+<comments>
```

Comment:
It would be enough with just one file (in the test enforce the validation or not depending on the test case).